dotnet/SK-dotnet.sln

```diff
@@ -525,6 +527,12 @@ Global
 		{13429BD6-4C4E-45EC-81AD-30BAC380AA60}.Publish|Any CPU.Build.0 = Debug|Any CPU
 		{13429BD6-4C4E-45EC-81AD-30BAC380AA60}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{13429BD6-4C4E-45EC-81AD-30BAC380AA60}.Release|Any CPU.Build.0 = Release|Any CPU
+		{8EE10EB0-A947-49CC-BCC1-18D93415B9E4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{8EE10EB0-A947-49CC-BCC1-18D93415B9E4}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{8EE10EB0-A947-49CC-BCC1-18D93415B9E4}.Publish|Any CPU.ActiveCfg = Debug|Any CPU
+		{8EE10EB0-A947-49CC-BCC1-18D93415B9E4}.Publish|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
Do we need the publish config?

Reply:
> Do we need the publish config?
The other examples are using the same pattern, I imagine this wont be a problem as soon the publish profiles are targetting `Debug|Any CPU` and won't be used in the release pipeline.